index.js

```diff
@@ -0,0 +1,105 @@
+// hasVowels (typeof Boolean): Returns true if the string contains vowels. 
+// This method implements Regular Expression.
+String.prototype.hasVowels = function () {
+    return /[aeiou]/i.test(this); 
+};
+
+// toUpper (typeof String): Returns the String in question but with all characters in upper
+// cases as applicable. This method should be implemented without the use of the
+// toUpperCase internal methods.
+
+String.prototype.toUpper = function () {
+    const check = /[a-z]/;
+    let result = '';
+        for(let i = 0; i<this.length; i++){
+        result += this[i].replace(check, String.fromCharCode(this[i].charCodeAt(0)-32));
+        }
+    return result;
+  };
+
+// toLower (typeof String): Returns the String in question but with all characters in their lowercases as applicable. This method should be implemented without the use of the
+// toLowerCase internal methods.
+
+String.prototype.toLower = function () {
+    const check = /[A-Z]/;
+    let result = '';
+        for(let i = 0; i<this.length; i++){
+        result += this[i].replace(check, String.fromCharCode(this[i].charCodeAt(0)+32));
```

Comment:
# Code Review - Nitpick
Looking at a line like this, it is not immediately clear what the code does to someone taking a look at it for the first time. The reader may have to parse the code mentally and try to evaluate the result. Adding a few comments or properly named loop-variables would be of great help to anyone reading the code afterward. See if you can refactor or clean it up.